web/src/app/v2/task/components/task-details-container/task-details-container.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { TaskFacadeService } from '@task/services/task-facade.service';
+import { debounceTime } from 'rxjs';
+
+@Component({
+  selector: 'taskana-task-details-container',
+  templateUrl: './task-details-container.component.html',
+  styleUrls: ['./task-details-container.component.scss']
+})
+export class TaskDetailsContainerComponent implements OnInit {
+  constructor(private route: ActivatedRoute, private taskFacade: TaskFacadeService) {}
+
+  ngOnInit(): void {
+    //TODO add takeuntil destroy, can use https://github.com/ngneat/until-destroy
```

Comment:
Implement `until-destroy` library in favor of manual component lifecycle destroying